back-end/routes/routes.js

```diff
@@ -20,6 +21,6 @@ module.exports = server => {
     server.route('/uploads').post(uploads);
     server.route('/collection').post(collection);
     server.route('/browse').get(fetchImages);
-    server.route('/collectionAdd').post(collection);
     server.route('/currentuser/').get(currentuser);
+    server.route('/addImageToCollection').post(collection);

```

Comment:
this is still going to `collection`  .  should be going to `addImageToCollection`